content/ecosystem/_index.html

```diff
@@ -0,0 +1,492 @@
+---
+title: RAPIDS.ai | Ecosystem
+description: RAPIDS libraries, frameworks, extensions and more
+---
+
+  <!-- Hero Shortcode -->
+  {{< hero hero-text="Ecosystem" button-link="#software" button-text="Go to Software" >}}
+
+  <!-- Intro -->
+  <section class="section mb-6">
+    
+    <div class="container">
+      <h1 class="has-text-centered mb-5"> An ecosystem of hardware, <br> software, and developers</h1>
+      <div class="columns">
+        <div class="column is-one-third">
+          <h2><i class="fa-regular fa-microchip"></i> Hardware </h2>
+          <p> RAPIDS runs on hardware powered by CUDA, the well-developed set of tools for GPU computing built by NVIDIA. Learn more about the different types of hardware and find the best fit for your use case. <a href="#hardware">Go To Section <i class="fa-solid fa-arrow-up-right"></i></a> </p>
+          
+        </div>
+        <div class="column is-one-third">
+          <h2><i class="fa-regular fa-code"></i> Software </h2>
+          <p> Built on a foundation of powerful hardware and parallelized by CUDA, the RAPIDS ecosystem extends from its core projects to more that extend its capabilities, add its features to their projects, or are built on top of RAPIDS. <a href="#software">Go To Section <i class="fa-solid fa-arrow-up-right"></i></a></p>
+
+          <br>
+          <img class="image img-center" src="/images/rapid-layers.png">
+        </div>
+        <div class="column is-one-third">
+          <h2><i class="fa-regular fa-laptop-code"></i>  Developers </h2>
+          <p> We hope you will consider integrating RAPIDS into your solution or workflow. RAPIDS is for developers ranging from competent python users to C++ experts. Build something great with RAPIDS. <a href="#developers">Go To Section <i class="fa-solid fa-arrow-up-right"></i></a></p>
+          
+        </div>
+      </div>
+    </div>
+
+  </section>
+
+  <!-- Hardware -->
+  <div id="hardware"></div>
```

Comment:
Having "hardware" appear as two different sections on the page is confusing. Is this bottom section actually helpful? Is it part of the ecosystem? If we had like a list of vendors and machines (which we shouldn't! that should be on corporate), I could see it but maybe just omit now.